data/json/effect_on_condition.json

```diff
@@ -1304,5 +1304,38 @@
       { "u_add_trait": "CHITIN2" },
       { "u_add_trait": "CRUSTACEAN_CARAPACE" }
     ]
+  },
+  {
+    "type": "effect_on_condition",
+    "//": "I couldn't for the life of me figure out how to have the item spawn open the place in container prompt, so I had to leave it as is.",
+    "id": "EOC_milk_yourself",
+    "condition": {
+      "u_compare_time_since_var": "milk_regeneration",
+      "type": "timer",
+      "context": "bio_processies",
+      "op": ">=",
+      "time": "12 h"
+    },
+    "effect": [
+      { "u_spawn_item": "milk_raw", "count": 40 },
```

Comment:
This should cost at least as many calories as the produced milk (5280 Calories). 1.2 * base metabolism over 12 hours is just 1080 extra Calories.

Reply:
How does a multiplayer of 5 sound?